Bugzilla/Comment/TagWeights.pm

```diff
@@ -39,7 +39,17 @@ use constant VALIDATORS => { };
 # There's no gain to caching these objects
 use constant USE_MEMCACHED => 0;
 
-sub tag    { return $_[0]->{'tag'} }
+###############################
+####      Accessors      ######
+###############################
+
+use Class::XSAccessor {
+    accessors => {
+        id  => __PACKAGE__->ID_FIELD,
+        tag => __PACKAGE__->NAME_FIELD,
```

Comment:
this should be name => __PACKAGE__->NAME_FIELD,

I wasn't sure of this so I wrote a test to exercise all this stuff, added to upasana-me/bmo#1.

(I hope this is okay. I sometimes contribute test cases to PRs when reviewing them)